src/views/task-list-page/task-list-workers-table-handler-icon/task-list-workers-table-handler-icon.tsx

```diff
@@ -0,0 +1,15 @@
+import { MdCheckCircleOutline, MdRadioButtonUnchecked } from 'react-icons/md';
+
+import { styled } from './task-list-workers-table-handler-icon.styles';
+
+export default function TaskListWorkersTableHandlerIcon(props: {
+  hasHandler: boolean;
+}) {
+  const Icon = props.hasHandler ? MdCheckCircleOutline : MdRadioButtonUnchecked;
+
+  return (
+    <styled.IconContainer $hasHandler={props.hasHandler}>
+      <Icon size="24px" display="block" />
```

Comment:
The icon sizes needs either to use theme

Reply:
Going to stick with theme.sizing as a starting point